tests/estimation/draft_estimator.py

```diff
@@ -0,0 +1,63 @@
+"""Simulation de données et application de quelques estimateurs
+
+-E
```

Comment:
What does this mean ?

Reply:
I wanted to run some estimators on small examples, but I shouldn't have commited this draft, it's not worth adding it